shower.cabal

```diff
@@ -1,5 +1,5 @@
 name: shower
-version: 0.2.0.1
+version: 0.2.0.2
```

Comment:
Since there are no code changes, I would avoid bumping the minor version and create a new revision instead.

Reply:
I don't trust the revisions mechanism anymore after seeing reports like https://github.com/monadfix/microlens/issues/112#issuecomment-495449076. I don't have a particularly strong opinion though.

Reply:
I see – `nix` can't deal with revisions, apparently. OK.